src/load.js

```diff
@@ -0,0 +1,108 @@
+import ToDo from './todo';
+import {
+  addTodoLs, getTodosls, updateTodosLs, deleteTodoLS, clearLs,
+} from './storage';
+import displayTodos from './display';
+import {
+  edit, updateDescriptionDom, hideEditFormDom, hideDeleteIconDom, hideEllipsyIconDom,
+} from './dom';
+
+const input = document.querySelector('#add-todo-input');
+const form = document.querySelector('#add-todo');
+const toDosArr = getTodosls();
+
+const loadEventListeners = () => {
+  window.addEventListener('DOMContentLoaded', displayTodos(toDosArr));
+
+  const checkboxes = document.querySelectorAll('.check');
+  const spans = document.querySelectorAll('.span-text');
+  const ellipsis = document.querySelectorAll('.ellipsis');
+
+  form.addEventListener('submit', () => {
+    const description = input.value;
+    const newTodo = new ToDo(description);
+    addTodoLs(newTodo);
+  });
+
+  checkboxes.forEach((check) => {
+    check.addEventListener('change', function () {
+      if (this.checked === true) {
+        const toDo = toDosArr[this.id];
+        ToDo.changeStatus(toDo);
+        updateTodosLs(toDo);
+      } else {
+        const toDo = toDosArr[this.id];
+        ToDo.changeStatus(toDo);
+        updateTodosLs(toDo);
+      }
+    });
+  });
+
+  spans.forEach((span) => {
+    span.addEventListener('click', (e) => {
+      const newSpan = document.createElement('span');
+      const editForm = document.createElement('form');
+      editForm.className = 'edit-form';
+      const inputEdit = document.createElement('input');
+      newSpan.appendChild(editForm);
+
+      edit(span, newSpan, editForm, inputEdit);
+      (e.target.parentElement).appendChild(newSpan);
+
+      editForm.addEventListener('submit', () => {
+        updateDescriptionDom(inputEdit, editForm, toDosArr);
+      });
+
+      if (newSpan) {
+        window.onclick = function (e) {
```

Comment:
:bulb: Here, it's best practice to target the closest element to where the event will occur. In this case, you should target the document instead of the whole window.